docs/user-guide/add-ai-assistant.md

```diff
@@ -0,0 +1,50 @@
+# Add Gen AI Assistant
+
+KubeRocketCI portal allows you to integrate Large Language Models (LLMs for short) to help you observe the overall platform status, as well as troubleshoot some issues related to the platform. Assistant's configuration in performed via the Configuration tab, in the Gen AI section.
+
+## Prerequisites
+
+This option requires you to have you AI assistant configured on a dedicated instance first. Thus, before starting the integration procedure, ensure to set up your Gen AI instance.
+
+## Set Up Integration
+
+To integrate the AI assistant, follow the steps below:
+
+1. Navigate to **KubeRocketCI** -> **Configuration** -> **Gen AI** and click the **+ Add link** button:
+
+  ![Add quick link](../assets/user-guide/add-link-for-gen-ai.png "Add quick link")
+
+2. Fill in the fields and click **Create**:
+
+  ![Set up quick link](../assets/user-guide/add-gen-ai-quick-link.png "Set up quick link")
+
+3. Navigate to **KubeRocketCI** -> **Configuration** -> **Gen AI** and click the **+ Add integration** button:
+
+  ![Integrate Gen AI](../assets/user-guide/add-gen-ai.png "Integrate Gen AI")
+
+4. Fill in the required fields:
+
+    * **Quick Link URL** - this link will be attached to the quick links presented on the KubeRocketCI portal main page. It should lead to the Gen AI instance endpoint.
+    * **API URL** - this is the URL address of the assistant's API server.
+    * **Assistant ID** - insert the ID of a dedicated assistant that will be used to interact with you.
+    * **Token** - this token will be used to get authorized access to you Gen AI instance. Ensure to set sufficient permissions so the platform will be able to interact with the assistant using this token.
+
+  ![Integrate Gen AI](../assets/user-guide/gen-ai-integration.png "Integrate Gen AI")
+
+5. Click the **Save** button.
+
+## Verify Integration
+
+As soon as the integration procedure is completed, click the AI chat button at the bottom-right corner of the screen:
+
+  ![AI chat window](../assets/user-guide/ai-chat-button.png "AI chat window")
+
+Type something in the chat to make sure the assistant works properly:
+
+  ![Ask assistant](../assets/user-guide/ask-assistant.png "Ask assistant")
+
+Besides, you will be able to select the **Integrate with CodeMie** option. This option enables Gen AI capabilities powered by the CodeMie tool for codebases:
```

Comment:
This is a separate integration. I don't see any description of it in the documentation